EasySettings/TemplateManager.cs

```diff
@@ -457,4 +482,41 @@ internal static AtlyssSpace CreateSpace(RectTransform container, RectTransform t
 
         return space;
     }
+
+    internal static AtlyssTextField CreateTextField(RectTransform container) => CreateTextField(container, TextFieldTemplate);
+
+    internal static AtlyssTextField CreateTextField(RectTransform container, AtlyssTextField template) => CreateTextField(container, template.Root);
+
+    internal static AtlyssTextField CreateTextField(RectTransform container, RectTransform template)
+    {
+        RectTransform root = Object.Instantiate(template, container);
+
+        List<Component> components = root.GetComponentInChildren<ComponentReferences>(true).components;
+
+        AtlyssTextField textField = new AtlyssTextField
+        {
+            Root = root,
+            Label = root.GetComponentsInChildren<Text>(true)[2], // couldnt get it to work with components[1]
+            Placeholder = ((InputField)components[0]).placeholder as Text,
```

Comment:
Move `GetComponentsInChildren()` out to cache it so we can also get the placeholder and "#" texts like mentioned above. Remove comment
```suggestion
        Text[] textComponents = root.GetComponentsInChildren<Text>(true);

        AtlyssTextField textField = new AtlyssTextField
        {
            Root = root,
            Label = textComponents[2],
            Placeholder = textComponents[0],
```